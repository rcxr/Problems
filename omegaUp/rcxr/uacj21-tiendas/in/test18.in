2
2
4
10 entrar centro-comercial
3 entrar d
3 entrar b
3 entrar a
