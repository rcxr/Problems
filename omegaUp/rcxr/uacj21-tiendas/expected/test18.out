a b centro-comercial d
